services/autorust/codegen/examples/gen_workspace.rs

```diff
@@ -19,7 +19,37 @@ fn main() -> Result<()> {
     let yml = CheckAllServicesYml { packages };
     yml.create("../../.github/workflows/check-all-services.yml")?;
 
-    // let yml = PublishServicesYml { packages };
-    // yml.create("../../.github/workflows/publish-services.yml")?;
+    // publish_sdks()?;
+    // publish_services()?;
```

Comment:
Can we put these behind a command line flag so that you're not expected to uncomment these to actually run the thing?

Reply:
I added a `publish` arg.